src/SFA.DAS.ApprenticeFeedback.Domain/Models/ExitSurvey/ExitSurveyAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace SFA.DAS.ApprenticeFeedback.Domain.Models.ExitSurvey
+{
+    public class ExitSurveyAttribute
```

Comment:
As discussed in the huddle just now, it might be a good idea to have this and the `FeedbackAttribute` class in Models\Feedback inherit from a common `SurveyAttribute` class, since they share the vast majority of their internal logic.

I would go about this by adding an `internal abstract class SurveyAttribute`,
renaming `FeedbackAttribute` to `FeedbackSurveyAttribute`,
then having both that and `ExitSurveyAttribute` derive from the abstract `SurveyAttribute` class.

Reply:
Have reworked this as discussed